google/cloud/ndb/model.py

```diff
@@ -1978,6 +1979,50 @@ def _deserialize(self, entity, p, unused_depth=1):
         """
         raise exceptions.NoLongerImplementedError()
 
+    def _legacy_deserialize(self, entity, p, unused_depth=1):
+        # Ported from legacy NDB, used for decoding pickle properties.
+        """Internal helper to deserialize this property from a protocol buffer.
```

Comment:
I think it would be helpful to note explicitly here that it's an older style (GAE) protocol buffer being deserialized here, and not the current Google Cloud Datastore protocol buffer.